packages/utils/regl-renderer/src/geometry-utils-V2/geom3ToGeometries.js

```diff
@@ -10,128 +10,134 @@ const {toArray} = require('../utils')
  * @param {Boolean} options.smoothLighting=false set to true if we want to use interpolated vertex normals
  * this creates nice round spheres but does not represent the shape of the actual model
  * @param {Float} options.normalThreshold=0.349066 threshold beyond which to split normals // 20 deg
- * @param {String} options.faceColor='#FF000' hex color
- * @returns {Object} {indices, positions, normals, colors}
+ * @param {String} options.color=[1, 0.4, 0, 1] default color of given geometry
+ * @returns {Object} [{indices, positions, normals, colors}, ...]
  */
 function geom3ToGeometries (listofgeom3, options) {
   const defaults = {
-    smoothLighting: false, // set to true if we want to use interpolated vertex normals this creates nice round spheres but does not represent the shape of the actual model
-    normalThreshold: 0.349066, // 20 deg
-    faceColor: [1, 0.4, 0, 1]// default color
+    smoothLighting: false,
```

Comment:
removing comments that add useful information is not the best  thing to do imho :)

Reply:
Those are the same as written in the documentation above, so I took out.

Reply:
whoops, very sorry, my bad !